content/tutorial/tutorial.md

```diff
@@ -450,7 +450,7 @@ class Board extends React.Component {
 
 [View the current code.](https://codepen.io/gaearon/pen/ybbQJX?editors=0010)
 
-We call `.slice()` to copy the `squares` array instead of mutating the existing array. Jump ahead a [section](/tutorial/tutorial.html#why-immutability-is-important) to learn why immutability is important.
+We call `.slice()` to copy the `squares` array instead of mutating the existing array. Jump ahead a [section](/content/tutorial/tutorial.md#why-immutability-is-important) to learn why immutability is important.
```

Comment:
It looks like you're reading these docs _in_ GitHub rather than on reactjs.org as intended 😄 

This link works for the website, linking one portion of text on [this page](https://reactjs.org/tutorial/tutorial.html) to a [different section](https://reactjs.org/tutorial/tutorial.html#why-immutability-is-important).